processor-tests/humans/ignore.txt

```diff
@@ -0,0 +1,57 @@
+# vim: set ft=sh :
+# that's for comments
+
+# This is borrowed from https://github.com/cormacrelf/citeproc-rs/blob/cd73f28945a980e984e73163f6f59e513336c570/crates/citeproc/tests/data/ignore.txt
+# It lists the test in this directory one should likely ignore for now, with
+# explanation.
+
+# relies on the "near-note": true, or "position": INTEGER, forcing options in
+# citeproc-js, # whereas in citeproc-rs cite positions only work automatically.
+position_NearNoteTrue.txt
+nameattr_EtAlSubsequentMinOnCitationInCitation.txt
+nameattr_EtAlSubsequentMinOnStyleInCitation.txt
+nameattr_EtAlSubsequentUseFirstOnCitationInCitation.txt
+nameattr_EtAlSubsequentUseFirstOnStyleInCitation.txt
+
+# these also use that, but the cites / clusters are in the correct order such that
+# the forcing options reflect the actual positions.
+# disambiguate_BasedOnSubsequentFormWithBackref.txt
+# disambiguate_BasedOnSubsequentFormWithLocator.txt
+
+# uses a whole lot of citeproc-js options
+form_ShortTitleOnly.txt
+
+# better to reject this kind of thing, isn't it?
+bugreports_EmptyInput.txt
+
+# note field parsing should be done higher up
+bugreports_IllustratorInExtra.txt
+
+# ABBREVIATIONS not implemented or part of CSL
+bugreports_TwoFullStops.txt
+variables_TitleShortOnAbbrevWithTitleGroup.txt
+variables_TitleShortOnAbbrevWithTitleCondition.txt
+number_EditionSort.txt
+variables_TitleShortOnAbbrevWithTitle.txt
+variables_ShortForm.txt
+
+# citeproc-js feature
+sort_CitationUnsorted.txt
+
+# BIBENTRIES
+api_UpdateItemsDeleteDecrementsByCiteDisambiguation.txt
+sort_CitationEdit.txt
+api_UpdateItemsReshuffle.txt
+api_UpdateItemsDelete.txt
+
+# bibliography-header mode
+bibheader_EntryspacingDefaultValueOne.txt
+bibheader_EntryspacingExplicitValueZero.txt
+bibheader_SecondFieldAlign.txt
+bibheader_SecondFieldAlignWithAuthor.txt
+bibheader_SecondFieldAlignWithNumber.txt
+
+# citation-rtf mode 
+bugreports_NonBreakingSpaceJoinFail.txt
```

Comment:
@fbennett can we indeed add these last two items to the ignore list, based on their reliance of undocumented modes?